xmtp_mls/src/groups/group_permissions.rs

```diff
@@ -1034,12 +1094,12 @@ impl PreconfiguredPolicies {
     }
 
     pub fn from_policy_set(policy_set: &PolicySet) -> Result<Self, PolicyError> {
```

Comment:
This function was causing the `Invalid Policy` Errors on existing groups after adding group image url. The reason was that the existing policies no longer matched the "default policies" for "Add Members" or "Admin only" because those policy sets now had one new property in them.